packages/contracts-bedrock/src/L1/L1StandardBridge.sol

```diff
@@ -114,11 +114,6 @@ contract L1StandardBridge is StandardBridge, ISemver {
         return superchainConfig.paused();
     }
 
-    /// @notice Allows EOAs to bridge ETH by sending directly to the bridge.
-    receive() external payable override onlyEOA {
```

Comment:
I am confused as to why you deleted the `receive` - we still want this in ether mode, but it should revert in custom gas token mode. I believe we already check downstream from here if using custom gas token so no code should be needed, just test coverage needed

Reply:
aah understood. thank you! will revert the changes + add the coverage needed